config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixclusters.yaml

```diff
@@ -245,6 +270,31 @@ spec:
                   you wish for communication with this endpoint not to be proxied,
                   please add the endpoint to the proxy spec.noProxy list.
                 properties:
+                  additionalTrustBundle:
+                    description: AdditionalTrustBundle is a PEM encoded x509 cert
+                      for the RootCA that was used to create the certificate for a
+                      Prism Central that uses certificates that were issued by a non-publicly
+                      trusted RootCA. The trust bundle is added to the cert pool used
+                      to authenticate the TLS connection to the Prism Central.
+                    properties:
+                      data:
+                        description: Data of the trust bundle if Kind is String.
+                        type: string
+                      kind:
+                        description: Kind of the Nutanix trust bundle
+                        enum:
+                        - String
```

Comment:
why do we need string or configmap? cant it be always string?

Reply:
The String type is for local testing / dev purposes. When it's running on a kubernetes cluster, it relies on the config map.